.github/workflows/windows-builds.yml

```diff
@@ -0,0 +1,212 @@
+        run: |
+          New-Item build64 -type directory -force
+          cd .\build64
+          cmake -G "${{ env.CMAKE_GENERATOR }}" -A "x64" -T "host=x64" -DCMAKE_TOOLCHAIN_FILE="${{ github.workspace }}\vcpkg\scripts\buildsystems\vcpkg.cmake" -DCMAKE_PREFIX_PATH="${{ runner.workspace }}\Qt64\Qt\5.12.9\msvc2017_64;${{ github.workspace }}\vcpkg\installed\x64-windows\bin" -DCMAKE_BUILD_TYPE="Release" -DWITH_SERVER=1 -DTEST=1 ..
+      - name: 'Build Cockatrice 64-bit'
+        working-directory: ${{ github.workspace }}
+        run: msbuild /m /p:Configuration=Release .\build64\Cockatrice.sln
+      - name: 'Build Cockatrice Installer Package 64-bit'
+        working-directory: ${{ github.workspace }}
+        run: |
+          cd build64
+          msbuild /m /p:Configuration=Release PACKAGE.vcxproj
+          cp *.exe ../Cockatrice-${{ env.GIT_TAG }}-${{ env.GIT_HASH }}-64bit-installer.exe
+      - name: 'Run Tests'
+        working-directory: ${{ github.workspace }}/build64
+        run: ctest -T Test -C Release
+      - name: 'Publish'
+        if: success()
+        uses: actions/upload-artifact@v2
+        with:
+          name: 'Cockatrice-${{ env.GIT_TAG }}-64bit'
+          path: './*.exe'
+  win32:
+    name: 'Windows 32-bit'
+    runs-on: [windows-latest]
+    env:
+      QT_VERSION: '5.12.9'
+      CMAKE_GENERATOR: "Visual Studio 16 2019"
+    steps:
+      - name: 'Add msbuild to PATH'
+        uses: microsoft/setup-msbuild@v1.0.2
+      - name: 'Checkout'
+        uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - name: 'Get Cockatrice git info'
+        shell: bash
+        working-directory: ${{ github.workspace }}
+        run: |
+          git fetch --prune --unshallow
+          echo "GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)" >> $GITHUB_ENV
+          echo "GIT_HASH=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
+          echo "GIT_TAG=$(git describe --tags --abbrev=0)" >> $GITHUB_ENV
+      - name: Restore Qt from cache
+        id: cache-qt
+        uses: actions/cache@v2
+        with:
+          path: |
+            ${{ runner.workspace }}/Qt32
+          key: ${{ runner.os }}-QtCache-32bit
+      - name: 'Install 32-bit Qt'
+        uses: jurplel/install-qt-action@v2
+        with:
+          cached: ${{ steps.cache-qt.outputs.cache-hit }}
+          version: '${{ env.QT_VERSION }}'
+          arch: 'win32_msvc2017'
+          dir: ${{ runner.workspace }}/Qt32
+      - name: 'Restore from cache and run vcpkg'
+        uses: lukka/run-vcpkg@v5
+        with:
+          vcpkgArguments: '@${{ github.workspace }}/vcpkg.txt'
+          vcpkgDirectory: '${{ github.workspace }}/vcpkg'
+          appendedCacheKey: ${{ hashFiles(env.vcpkgResponseFile) }}
+      - name: 'Configure Cockatrice 32-bit'
+        working-directory: ${{ github.workspace }}
+        run: |
+          New-Item build32 -type directory -force
+          cd .\build32
```

Comment:
```suggestion
          cd build32
```

Reply:
Whoops, thought I removed this already. Looks like I did in the next step. Fixed.